src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.Tests/AdminServiceTest.cs

```diff
@@ -58,78 +58,88 @@ public class AdminServiceTest
 
         public AdminServiceTest()
         {
-            _adminRepositoryMock = new Mock<IAdminRepository>();
-            _baseTestRepositoryMock = new Mock<IBaseTestRepository>();
+            this._adminRepositoryMock = new Mock<IAdminRepository>();
+            this._baseTestRepositoryMock = new Mock<IBaseTestRepository>();
         }
 
         [Fact]
-        public void GetAllTests_ReturnTestDtoList()
+        public void GetAllTestsReturnTestDtoList()
```

Comment:
please disable the rule, that doesn't allow the underscore in method names and revert this changes  (disable rule only in AnalysersTest.ruleset)